balta/src/test/scala/za/co/absa/db/balta/classes/QueryResultRowIntegrationTests.scala

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright 2023 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.db.balta.classes
+
+import org.scalatest.funsuite.AnyFunSuiteLike
+import za.co.absa.db.balta.testing.classes.DBTestingConnection
+
+import java.sql.{Date, ResultSetMetaData, Time}
+import java.text.SimpleDateFormat
+import java.time.{Instant, LocalDateTime, OffsetDateTime}
+import java.util.UUID
+
+class QueryResultRowIntegrationTests extends AnyFunSuiteLike with DBTestingConnection{
+  private val (tableRows: List[QueryResultRow], metadata: ResultSetMetaData) = DBTable("testing.base_types").all("long_type") { q =>
+    (q.toList, q.resultSetMetaData)
+  }
+
+  test("fieldNamesFromMetada") {
+    val result = QueryResultRow.fieldNamesFromMetdata(metadata)
+
+    val expecedResult = Seq(
+      "long_type",
+      "boolean_type",
+      "char_type",
+      "string_type",
+      "int_type",
+      "double_type",
+      "float_type",
+      "bigdecimal_type",
+      "date_type",
+      "time_type",
+      "timestamp_type",
+      "timestamptz_type",
+      "uuid_type",
+      "array_int_type")
```

Comment:
this formatting is killing me :D

Reply:
What exactly? :-S

Reply:
```scala
    val expecedResult = Seq(
      "long_type",
      "boolean_type",
      "char_type",
      "string_type",
      "int_type",
      "double_type",
      "float_type",
      "bigdecimal_type",
      "date_type",
      "time_type",
      "timestamp_type",
      "timestamptz_type",
      "uuid_type",
      "array_int_type"
   )
```

i.e. parenthesis aligned with each other

Reply:
but I don't want to deal with these things as this is already an overcame problem with correctly set auto formatters, which you hate :(